crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -806,8 +806,7 @@ where
             );
 
             return Err(Error::StringError(format!(
-                "Request limit ({}) exceed the server limit: {} ",
-                limit, MAX_SEGMENT_HEADERS_PER_REQUEST
+                "Request limit ({limit}) exceed the server limit: {MAX_SEGMENT_HEADERS_PER_REQUEST} "
```

Comment:
This doesn't need to be fixed in this PR, but there's an odd extra space at the end of this log message. A search and replace could fix them all at once.